cli/functionary/config.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+import click
+from dotenv import get_key, set_key
+
+
+def save_config_value(key, value):
+    functionary_dir = Path.home() / ".functionary"
+    if not functionary_dir.exists():
+        functionary_dir.mkdir()
+
+    config_file = functionary_dir / "config"
+    set_key(
+        config_file,
+        key,
+        value,
+        quote_mode="always",
+        export=False,
+        encoding="utf-8",
```

Comment:
It doesn't hurt anything the way you have it, but when you see this function documented as:

```python
set_key(
    dotenv_path,
    key_to_set,
    value_to_set,
    quote_mode="always",
    export=False,
    encoding="utf-8",
)
```

Those last 3 have default values set, meaning they are optional parameters.  Since you're just using the default values, you could have left them off and just made this `set_key(config_file, key, value)`.